### PR TITLE
Update faq-md-app-guard.md

### PR DESCRIPTION
**Pull request by sazankha**
Added a known issue and it's mitigation

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md [hunk 1]
```diff
@@ -159,3 +159,13 @@ Step 2:
 3. Disable IPNAT (Optional): 
 `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\IPNat\Start = 4`.
 4. Restart the device.
+
+### Why doesn't Application guard work, even though it's enabled through Group Policy?
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md:163]:
```suggestion
### Why doesn't Application Guard work, even though it's enabled through Group Policy?
```

File: windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md [hunk 2]
```diff
@@ -159,3 +159,13 @@ Step 2:
...
+
+Application Guard must meet all these prerequisites to be enabled in enterprise mode: [System requirements for Microsoft Defender Application Guard](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md:165]:
```suggestion
Application Guard must meet all these prerequisites to be enabled in Enterprise mode: [System requirements for Microsoft Defender Application Guard](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). 
```

File: windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md [hunk 3]
```diff
@@ -159,3 +159,13 @@ Step 2:
 3. Disable IPNAT (Optional): 
 `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\IPNat\Start = 4`.
 4. Restart the device.
+
+### Why doesn't Application guard work, even though it's enabled through Group Policy?
+
+Application Guard must meet all these prerequisites to be enabled in enterprise mode: [System requirements for Microsoft Defender Application Guard](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). 
+To understand why it is not enabled in enterprise mode, check the status of the evaluation to understand what's missing.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md:166]:
```suggestion
To understand why it is not enabled in Enterprise mode, check the status of the evaluation to understand what's missing.
```

File: windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md [hunk 4]
```diff
@@ -159,3 +159,13 @@ Step 2:
...
+
+For CSP (Intune) you can query the status node by using **Get**. This is described in the [Application Guard CSP](https://docs.microsoft.com/windows/client-management/mdm/windowsdefenderapplicationguard-csp). 
+In this page you will see the **status** node as well as the meaning of each bit.  If the status is not 63, you are missing a prerequisite.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-application-guard/faq-md-app-guard.md:169]:
```suggestion
On this page, you will see the **status** node as well as the meaning of each bit.  If the status is not 63, you are missing a prerequisite.
```

**Comment by e0i** [comment]
@Kateyanne This one has been approved by its author and can be merged. Thank you.

**Pull request merged**